cfgov/agreements/management/commands/import_agreements.py

```diff
@@ -1,10 +1,12 @@
 import os
-import urllib
-from datetime import datetime
+from zipfile import ZipFile
+
+import fnmatch
 
 from django.core.management.base import BaseCommand
 
-from .util import clear_tables, update_agreement, update_issuer, upload_to_s3
+from agreements.management.commands import util
+from agreements.models import Issuer, Agreement
 
 
 class Command(BaseCommand):
```

Comment:
Could you update the command's docstring to reflect that source is a zip file, not csv?

Reply:
Latest commit addresses this